ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -36,44 +36,82 @@ Item {
     signal openProfileClicked()
     signal openAppSearch()
 
-    Component.onCompleted: {
-        appMain.openContactsPopup.connect(function(){
-            Global.openPopup(contactRequestsPopup, {chatSectionModule})
-        })
-    }
```

Comment:
Trying to figure out what this code used to do... Am I getting it right that this would open the contact requests popup as soon as the contacts column view is done loading?

If this is a "yes", do we somehow preserve that behaviour after those changes?

Reply:
Yeah, part of the obsolete signals removal; it was trying to connect to the `openContactsPopup()` signal (in a very non-declarative way and reaching out of context for `appMain`) and open the contact request popup. However, that signal is never emitted anywhere, and then there's this piece of code at the end of this file:
```qml
    Connections {
        target: root.store.mainModuleInst

        function onOpenContactRequestsPopup() {
            Global.openPopup(contactRequestsPopup)
        }
    }
```
which does exactly that (as can be seen on the attached screenshot)

Reply:
Thank you for clarifying!